cli/src/migrateMongo/index.js

```diff
@@ -80,8 +81,32 @@ const getOutstandingMigrations = async ({ down, up, migrations }) => {
   return outstandingMigrations;
 };
 
+export const fixOrderFn = async ({ migrations }) => {
```

Comment:
I think we should rename this to `fixDbMigrationOrder` to make this a little clearer to a reader. Might be worth putting a link to the original Github issue in a comment above this to give it some context.